pyperf/_psutil_memory.py

```diff
@@ -0,0 +1,44 @@
+try:
+    import os
+    import psutil
+    import threading
+    import time
+except ImportError:
+    raise ImportError('psutil is not installed')
+
+
+class PeakMemoryUsageThread(threading.Thread):
+    def __init__(self):
+        threading.Thread.__init__(self)
+        self.process = psutil.Process(os.getpid())
```

Comment:
I'm not used to the psutils.Process API. Is there a close() method when you are done with it?

Reply:
AFAIK, no close method for this API

Reply:
Ok, I was just curious.